content/webapp/services/prismic/transformers/body.ts

```diff
@@ -75,6 +73,7 @@ import { Props as TitledTextListProps } from '@weco/content/components/TitledTex
 import { Props as AsyncSearchResultsProps } from '@weco/content/components/SearchResults/AsyncSearchResults';
 import { Venue } from '@weco/common/model/opening-hours';
 
+// TODO all the TODOs in this file
```

Comment:
😄 what's this/should it be addressed as part of this?

Reply:
oh yeah, I think I've the ones I cared about, so I'll remove this comment. It was just a reminder to self. There's 2 TODOs that David added that concern labels, but they can be done separately and another one that predates the slice machine work, which again can be addressed separately.